packages/gatsby-cli/src/init-starter.js

```diff
@@ -154,6 +155,13 @@ module.exports = async (starter: string, options: InitOptions = {}) => {
     return
   }
 
+  if (!isValid(rootPath)) {
+    report.panic(
+      `Could not create a project called ${rootPath} because of npm naming restrictions`
```

Comment:
I think the message need to be adjusted a bit - for example:
```suggestion
      `Could not create a project in ${rootPath} because it's not valid path`
```

Reply:
:+1: